_examples/aks_with_microsoft_entra_id/example.tf

```diff
@@ -99,14 +99,7 @@ module "aks" {
 
   kubernetes_version      = "1.27.7"
   private_cluster_enabled = false
-  default_node_pool = {
```

Comment:
Why are we removing this?
Pls don't remove this. 
do same in all other examples also. Also once you have re-added this pls test also.